LongHeaderUnitFinder/LongHeaderUnitFinder.vcxproj

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
The new sample projects don't have their output directory set to the correct location. The output directory should be set to `$(SolutionDir)out\$(Platform)\$(Configuration)\$(ProjectName)\` so that the executables and DLLs get copied to, e.g., `out\x64\Release\LongHeaderUnitFinder`. You can set those properties in Visual Studio in the General tab of the project properties dialog.

Reply:
Fixed.